gdal_subsetter/utilities.py

```diff
@@ -57,41 +56,69 @@ def get_version() -> str:
     return version
 
 
-def get_files_from_unzipfiles(extract_dir: str, filetype: str,
-                              variables=None) -> List[str]:
+def has_world_file(file_mime_type: str) -> bool:
+    """ Determine if the given MIME type for a transformed output is expected
+        to have an accompanying ESRI world file.
+
     """
-    inputs: extract_dir which include geotiff files, filetype is
-    either 'tif' or 'nc', variables is the list of variable names.
-    return: filelist for variables.
+    return any(world_mime in file_mime_type.lower()
+               for world_mime in ['png', 'jpeg'])
```

Comment:
I'm not sure I think this is the best way to check this - for example we could just do:

```
return file_mime_type.lower() in ('image/png', 'image/jpeg')
```

As a first step, though, I just migrated the current check to this function, with a view to later clean-up.

Reply:
no strong opinions.  This is fine for now